contents/docs/data/persons.mdx

```diff
@@ -16,18 +16,21 @@ export const personProfileDark = "https://res.cloudinary.com/dmukukwp6/image/upl
     className="rounded shadow-xl"
 />
 
-People in PostHog represent users who do events.
+People in PostHog represent the users behind your events. You can view them in the [People tab](https://us.posthog.com/persons).
 
-Every event has a `distinct_id` representing a person attached to it. However, not every `distinct_id` has an associated person profile.
+People have [user properties](/docs/product-analytics/user-properties), which enables you do things like:
 
-Capturing person profiles and user properties enables:
+- Filter on user properties
+- Create [cohorts](/docs/data/cohorts)
+- Use user properties for targeting feature flags, A/B tests, and surveys.
+- Track initial UTM values and referrers across anonymous and [identified](/docs/product-analytics/identify) users.
 
-- Viewing person profiles in the [People tab](https://us.posthog.com/persons)
-- Filtering on user properties
-- Merging identities across `distinct_id` values (i.e. merging an anonymous user with a logged in user)
-- Cohort creation
-- Feature flag, experimentation, and survey targeting on user properties
-- Tracking initial UTM values and referrers across anonymous and identified users
+## People and person profiles
+
+Using people in PostHog requires capturing events with **person profiles**. Person profiles are how PostHog attributes events and properties to specific users. Examples of scenarios where you want to capture events with person profiles are:
+
+- Tracking logged-in users in B2B and B2C SaaS apps
+- Low volume API server events
```

Comment:
I don't full understand why low volume would change something, you could just not have very many users.

Reply:
Alternatives:

- Growth and marketing teams wanting to analyze the complete conversion lifecycle
- Doing user segmented product analysis